site/gatsby-site/src/components/discover/filterTypes/Classifications.js

```diff
@@ -0,0 +1,259 @@
+        role="button"
+        tabIndex={0}
+        onClick={() => refine(item.value)}
+        data-cy={item.value}
+      >
+        <div>
+          <b>{namespace}</b> : {name} : {item.label}
+        </div>
+        <FontAwesomeIcon icon={faClose} />
+      </div>
+    ));
+}
+
+function Search({ setSearchResults }) {
+  const { t } = useTranslation();
+
+  const {
+    items: searchItems,
+    searchForItems,
+    isFromSearch,
+  } = useRefinementList({ attribute: 'classifications' });
+
+  const [query, setQuery] = useState('');
+
+  const debouncedSearchForItems = useCallback(
+    debounce((query) => {
+      searchForItems(query);
+    }, 1000),
+    []
+  );
+
+  useEffect(() => {
+    if (query.length > 2) {
+      debouncedSearchForItems(query);
+    }
+
+    if (query.length == 0) {
+      setSearchResults(null);
+    }
+  }, [query]);
+
+  useEffect(() => {
+    if (isFromSearch) {
+      setSearchResults(searchItems);
+    }
+  }, [searchItems, isFromSearch]);
+
+  return (
+    <div className="mt-2" data-cy="search">
+      <TextInputGroup
+        name="query"
+        label={'Search'}
+        placeholder={t('Search')}
+        schema={null}
+        icon={null}
+        type="search"
+        values={{ query }}
+        errors={{}}
+        touched={{}}
+        handleChange={(e) => setQuery(e.currentTarget.value)}
+      />
+    </div>
+  );
+}
+
+export default function Classifications({ taxa }) {
+  const { indexUiState } = useInstantSearch();
+
+  const { items: namespaces } = useRefinementList({ attribute: 'namespaces' });
+
+  const selectedAttributes = Object.keys(indexUiState.refinementList).filter((key) => {
+    return taxa.map((t) => t.namespace).some((n) => key.startsWith(`${n}.`));
+  });
+
+  const [searchResults, setSearchResults] = useState(null);
+
+  return (
+    <div data-cy="classifications">
+      <div className="space-y-1" data-cy="selected-refinements">
+        {selectedAttributes.map((attribute) => {
+          return <SelectedRefinement key={attribute} attribute={attribute} />;
+        })}
+      </div>
+
+      <Search setSearchResults={setSearchResults} />
+
+      {searchResults == null || searchResults.length > 0 ? (
+        <div className="mt-3 space-y-2">
+          {taxa.map((taxonomy) => {
+            const refinement = namespaces.find((r) => r.value === taxonomy.namespace);
+
+            if (!refinement) {
+              return <EmptyNamespace key={taxonomy.namespace} taxonomy={taxonomy} />;
+            }
+
+            return (
+              <Namespace
+                key={taxonomy.namespace}
+                taxonomy={taxonomy}
+                refinement={refinement}
+                searchResults={searchResults}
+              />
+            );
+          })}
+        </div>
+      ) : (
+        <div className="mt-3">No results</div>
+      )}
+    </div>
+  );
+}
+
+export const touchedCount = ({ searchState }) => {
```

Comment:
unused? (or I can't find it in code search, my bad)

Reply:
Those are used here:
https://github.com/responsible-ai-collaborative/aiid/blob/ed22c006d03a36b3752fedc17b6ce48f7bb43733/site/gatsby-site/src/components/discover/Filter.js#L144

Reply:
Thanks, I figured I was likely missing it.